src/frontend/src/components/mainWindow.ts

```diff
@@ -0,0 +1,53 @@
+import { html, TemplateResult } from "lit-html";
+import { ifDefined } from "lit/directives/if-defined.js";
+import { icLogo } from "./icons";
+import { footer } from "./footer";
+
+/**
+ * main window template
+ *
+ * To avoid having to repeat the same structure in every page,
+ * we use this component to wrap the content of each page.
+ * This way, we can change the structure of the main window
+ * in one place.
+ *
+ * Before it was just a DIV with a class, but now its a more complex
+ * component that includes the logo, the footer, and the container.
+ *
```

Comment:
```suggestion
 * It is a component that includes the logo, the footer, and the container.
 *
```

After this is merged, no one will care what it was before